src/main/tools/linux-sandbox-pid1.cc

```diff
@@ -297,23 +297,10 @@ static void MountFilesystems() {
     }
   }
 
-  if (mount(opt.working_dir.c_str(), opt.working_dir.c_str(), nullptr, MS_BIND,
-            nullptr) < 0) {
-    DIE("mount(%s, %s, nullptr, MS_BIND, nullptr)", opt.working_dir.c_str(),
-        opt.working_dir.c_str());
-  }
-
   std::unordered_set<std::string> bind_mount_sources;
-
   for (size_t i = 0; i < opt.bind_mount_sources.size(); i++) {
     const std::string &source = opt.bind_mount_sources.at(i);
     bind_mount_sources.insert(source);
-    const std::string &target = opt.bind_mount_targets.at(i);
```

Comment:
Won't moving this down below the `writable_file` handling cause writable files that occur in `opt.bind_mount_sources` to not be mounted writably?

Reply:
Do those exist? (I don't know, I'm not familiar with this code, all I know is that the tests pass)